share/templates/07-Band_and_Percussion/03-Brass_Band/03-Brass_Band.mscx

```diff
@@ -700,17 +700,17 @@
           </StaffType>
         </Staff>
       <trackName>2nd Horn</trackName>
-      <Instrument id="c-horn-alto">
+       <Instrument id="eb-alto-horn">
```

Comment:
And here an extra space got added. (Not a big deal, but still nice to keep everything as clean as possible)
```suggestion
      <Instrument id="eb-alto-horn">
```